src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -314,6 +308,7 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       );
       break;
     case STATUS_TYPES.remand: {
+      // TODO: should we break this out into its own template/component?
```

Comment:
Good call. Mark and I plan on doing a lot of refactoring work to this app when we get the chance. This is a good candidate for that. For now, I think we leave it alone, but please leave this comment; we're planning on searching for `TODO`s. :smile:

Reply:
Thanks for the context @cvalarida.